config/e3sm/machines/config_machines.xml

```diff
@@ -2837,8 +2837,8 @@
 </machine>
 
 <default_run_suffix>
-  <default_run_exe>${EXEROOT}/acme.exe </default_run_exe>
-  <default_run_misc_suffix> >> acme.log.$LID 2>&amp;1 </default_run_misc_suffix>
+  <default_run_exe>${EXEROOT}/e3sm.exe </default_run_exe>
```

Comment:
I don't remember why this was separate.  I think you could do ${EXEROOT}/${MODEL}.exe and
${MODEL}.log.$LID

Reply:
I'd rather do that in a separate PR.